minerl/Malmo/Minecraft/run/config/forge.cfg

```diff
@@ -65,7 +65,7 @@ D:zombieBaseSummonChance=0.1
 D:zombieBabyChance=0.05
 
 # Enable the forge block rendering pipeline - fixes the lighting of custom models.
-B:forgeLightPipelineEnabled=true
+B:forgeLightPipelineEnabled=false
```

Comment:
what does this do?

Reply:
This enables custom forge lighting which we don't want